src/main/java/com/google/devtools/build/lib/exec/SpawnLogContext.java

```diff
@@ -138,6 +138,7 @@ public void logSpawn(
     if (!timeout.isZero()) {
       builder.setTimeoutMillis(timeout.toMillis());
     }
+    // I'm not sure whether to use Spawns.mayBeCached or Spawns.mayBeCachedRemotely here.
```

Comment:
@ishikhman thoughts on this? I'm leaning towards leaving it as-is since `mayBeCached` is more restrictive than `mayBeCachedRemotely`

Reply:
if we make them the same (see my comment above) - it does not really matter for now. From the logical point of view I'd agree with `mayBeCached` :)

Reply:
Still not sure what to do here since we decided to keep the `Spawns.mayBeCached` and `Spans.mayBeCachedRemotely` separate.

`Spawns.mayBeCached` is more restrictive, so I think it makes sense to use that (even if `Spanws.mayBeCachedRemotely` is false, the result can still be cached if a local cache is used; however, if `Spawns.mayBeCached` is false, then the result may never be cached). Thoughts on this?

Reply:
i'd say, lets do as you suggest - keep `Spawns.mayBeCached` here. Ideally we would have to split this log field into 2: one for local cache, one - for remote, but for now we can keep the `old` behavior.

I believe this is somehow related to https://github.com/bazelbuild/bazel/issues/8192.

Reply:
Sounds good. Yeah, that's definitely related; I think keeping the current behavior for now is the right call though.